VAMobile/documentation/docs/Engineering/FrontEnd/DevelopmentBestPractices.md

```diff
@@ -0,0 +1,130 @@
+
+Each pull request moves through a series of steps.
+
+1. Creation: an engineer creates a PR. See "Pull request creation" below.
+2. Review: a different engineer reviews the PR. See "Pull request review" below.
+3. Approval: when the reviewer is satisfied, they approve the PR and change the pipeline to "With QA (pre-develop)" so QA knows to begin testing.
+4. QA: a QA engineer tests the branch and updates TestRail. QA informs the PR creator of any issues they find. On approval, QA comments in the ticket and changes the pipeline to "Ready to merge to develop".
+5. Merge: the PR creator hits "Merge pull request" to merge their branch into the develop branch. On the ticket, the PR creator changes the pipeline to "With QA (develop)" and selects the current release under "Releases".
+6. Post-merge: QA completes final testing and closes the ticket.
+
+### Pull request creation
+
+When creating a PR, fill out each section in the PR template, including:
+
+- Description: provide context so a code reviewer or future developer understands why the PR exists and what it does
+- Screenshots: add before/after screenshots or videos showing the changes
+- Testing information: confirm testing on iOS and Android. Document any special considerations like edge cases or areas of focus for QA
+- Reviewer validations: tell reviewers what to check. You can usually copy the Acceptance Criteria from the ticket
+- Complete all PR checklist items
+
+:::info
+If you're actively working on your branch, create a draft PR. Move the PR out of draft status when it's complete and ready for review.
+:::
+
+Immediately after creating the PR, click the "Connect issue" button and choose the associated ticket. You must be logged into ZenHub to see the "Connect issue" button.
+
+:::important
+It's critical to link the PR to the ticket with the "Connect issue" button so we can track the work performed.
+:::
+
+If your PR is especially complex, you can annotate the PR with comments to help reviewers understand why you made particular decisions.
+
+When you create a PR, GitHub Actions automatically runs linting checks, unit tests, and E2E tests on your branch. Fix any failures, and make sure unit and E2E tests are modified or extended to cover your changes. See our [Unit Tests documentation](/docs/Engineering/FrontEnd/Testing/UnitTesting) for best practices.
+
+### Pull request review
+
+At least one review is required on every PR. Here are some guidelines for creating and reviewing PRs.
+
+As a PR creator:
+
+- Try to keep PRs under 400 lines. The larger the PR, the more difficult it is to review thoroughly.
+- If a reviewer asks questions, add a comment under each question with your answer.
+- If a reviewer requests changes, implement them or explain why you disagree.
+- If no one reviews your PR within 24 hours, request a review on Slack.
+
+As a PR reviewer:
+
+- Provide constructive feedback in a calm, civil manner. We're all working together to make the app better.
```

Comment:
(╯°□°)╯︵ ┻━┻

Reply:
I would also like us to largely keep feedback targeted towards best practice and away from opinion. Also, if you're not officially requesting changes to a PR you can say something like "leaving approval to (person) because I don't know enough about this code" or approve the PR. I don't want people dropping opinions in comments and expecting a back and forth if changes aren't being requested

Reply:
Updated to add these points